src/components/App/App.js

```diff
@@ -1,26 +1,37 @@
 import React from 'react';
 import { BrowserRouter as Router, Route } from 'react-router-dom';
 import { PerseidsHeader, PerseidsFooter } from 'perseids-react-components';
-
+import I18n from '../I18n';
 import 'perseids-react-components/build/css/index.css';
 
 import AsyncLookup from '../AsyncLookup';
 
+const base = '/:locale(en|fr)?';
 const App = () => (
   <React.Fragment>
-    <PerseidsHeader>
-      LSJ Lexicon
-    </PerseidsHeader>
+    <Route
```

Comment:
You'll need to move the `Router` component so that these `Route`s are inside it. Otherwise, you'll get a `You should not use <Route> or withRouter() outside a <Router>` error.